modules/table/src/main/java/org/apache/ignite/internal/table/distributed/TableManager.java

```diff
@@ -3013,7 +3038,7 @@ private CompletableFuture<Void> startTables(long recoveryRevision, @Nullable Hyb
                     if (throwable != null) {
                         LOG.error("Error starting tables", throwable);
                     } else {
-                        LOG.debug("Tables started successfully");
+                        LOG.info("Tables started successfully [count={}]", startTableFutures.size());
```

Comment:
I'm a little worried that I might have missed something. Are there any important changes in TableManager besides removing listeners?

Reply:
Re-reviewed it once more time. It's just listeners registration/deregistration, formatting issues and this logging change to ease diagnosing